recipes/analisi/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "analisi" %}
+{% set version = "0.1.8" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rikigigi/analisi/archive/v{{ version }}.tar.gz
+  sha256: 13a1f025723e96d84a354761b4bcc4afdad30ef5abc3397e63792410efa7fa6d
+
+build:
+  number: {{ build }}
+  skip: True  # [not linux]
+  string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ mpi }}_{{ build }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - python
```

Comment:
```suggestion
```

Reply:
there is a python interface that needs python to be compiled, isn't python a build dependency?

Reply:
That means `python` has to be in `host` as you need Python for the target system that you're building for.

A build dependency is purely for actually building the code like `cmake`, `make` or the compiler itself. Some packages do have Python scripts as part of their build system however it's quite rare.

There is a nice post about how conda-forge uses this for cross-compiling here: https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/

Reply:
Ok, thanks, now it is more clear